recipe/meta.yaml

```diff
@@ -36,7 +36,7 @@ outputs:
         - ipython
         - jupyter
         - matplotlib >2.2.2
-        - numpy >1.13,!=1.17
+        - numpy >1.13,!=1.17.*
```

Comment:
Wouldn't ``<1.17`` be better since it would be future-proof?

Reply:
Yes, I was testing this more or less out of curiosity!